src/queries/useLinkSuggestions/useLinkSuggestions.js

```diff
@@ -6,13 +6,17 @@ const useLinkSuggestions = () => {
   const ky = useOkapiKy();
 
   const { mutateAsync, isLoading } = useMutation(
-    ({ body, isSearchByAuthorityId }) => {
+    ({ body, isSearchByAuthorityId, ignoreAutoLinkingEnabled }) => {
       let api = 'records-editor/links/suggestion';
 
       if (isSearchByAuthorityId) {
         api += '?authoritySearchParameter=ID';
       }
 
+      if (ignoreAutoLinkingEnabled) {
```

Comment:
Let's use `querystring` or `URLSearchParams` to build query to make it a little more readable

Reply:
Thanks, fixed.